### ISSUE TITLE
Create dataset loader for EU-ADR Corpus

### ISSUE THREAD

**Issue by jason-fries** [bigscience-workshop/biomedical#248]
## Adding a Dataset
- **Name:** EU-ADR Corpus
- **Description:** *None provided*
- **Task:** NER|RE
- **Paper:** https://www.sciencedirect.com/science/article/pii/S1532046412000573
- **Data:** https://biosemantics.erasmusmc.nl/downloads/euadr.tgz
- **License:** ?

**Comment by giganttheo** [comment]
#self-assign

**Comment by hakunanatasha** [comment]
@giganttheo let us know how this is going; if your bandwidth is busy, we can free this up as I believe you're working on a few datasets already!

**Comment by giganttheo** [comment]
Hi @hakunanatasha. I was focusing on other datasets first but I will take a look at this one in a few days 😀

**Comment by giganttheo** [comment]
The linked data is lacking the corpus information. It contains all the annotations but not the corpus that is annotated.

I have found this repo that contains some of this dataset ( https://github.com/EsmaeilNourani/Deep-GDAE/tree/master/data ) but not all of it it seems (355 sentences). So I have worked with that for now and will be able to suggest a loading script with this.
But I struggle to find the complete corpus for the whole dataset that goes with the annotations :/
